vendor/bat-native-ads/resources/bat_ads_resources.grd

```diff
@@ -11,9 +11,9 @@
       <include name="IDR_ADS_CATALOG_SCHEMA" file="catalog-schema.json" type="BINDATA" />
       <include name="IDR_ADS_BUNDLE_SCHEMA" file="bundle-schema.json" type="BINDATA" />
       <include name="IDR_ADS_SAMPLE_BUNDLE" file="sample_bundle.json" type="BINDATA" />
```

Comment:
I think we can add `gzip` flag here as well. They are not 8Mb files, but it will save us some space.